source/includes/harvest/_custom_fields.md

```diff
@@ -288,30 +333,50 @@ department_ids | Array of integers | no | If included, this custom field is only
 | custom_field_options.external_id | string | no | The external_id for the custom field. Used for integrating with external HRIS.
 | generate_email_token | boolean | no | If this is sent as true, a default `template_token_string` will be generated for the new Custom Field.
 
-> The above returns a success message on success with a 201 response.
 
->  If successful, the above command returns a 201 response code along with a JSON representation of the newly created custom field. The JSON structure looks like this:
+## PATCH: Update Custom Field
+
+This endpoint updates an existing custom field.
+
+```shell
+curl -X PATCH 'https://harvest.greenhouse.io/v1/custom_fields/{id}'
+-H "On-Behalf-Of: {greenhouse user ID}"
+-H "Authorization: Basic MGQwMzFkeODyN2VhZmEMWRjMzc1YZjMqmUwNjsdlMjQ6"
+```
+
+> The above command takes a JSON request, structured like this:
+
+```json
+{
+  "name": "New Name",
+  "description": "New description",
+  "private": false,
+  "template_token_string": "token123"
+}
+```
+
+> If the update is successful, the above command returns JSON structured like this:
 
 ```json
 [
   {
     "id": 123456,
-    "name": "Custom Field Name",
+    "name": "New Name",
     "active": true,
     "field_type": "job",
     "priority": 1,
     "value_type": "single_select",
-    "private": true,
+    "private": false,
     "required": false,
     "require_approval": true,
     "trigger_new_version": false,
     "name_key": "custom_field_name",
-    "description": "Field description",
+    "description": "New description",
     "expose_in_job_board_api": false,
     "api_only": false,
     "offices": [],
     "departments": [],
-    "template_token_string": "{{FIELD1}}",
+    "template_token_string": "{{token123}}",
```

Comment:
I liked using an all-caps token, since it would make it consistent with the defaults set by our system.

Reply:
Ah, that's a good point.  I will update it